palace/linalg/petsc.cpp

```diff
@@ -5,7 +5,7 @@
 
 #include <petsc.h>
 #include <petscblaslapack.h>
-#include <general/forall.hpp>
+#include <mfem/general/forall.hpp>
 // #include "linalg/hypre.hpp"
```

Comment:
delete?

Reply:
This file is going to get removed very shortly as the PETSc wrapper classes get removed so I'll leave it for now.